packages/node-client/lib/index.ts

```diff
@@ -236,13 +242,24 @@ export class Nango {
         providerConfigKey: string,
         connectionId: string,
         forceRefresh?: boolean
-    ): Promise<string | OAuth1Token | BasicApiCredentials | ApiKeyCredentials | AppCredentials> {
+    ): Promise<
+        | string
+        | OAuth1Token
+        | BasicApiCredentials
+        | ApiKeyCredentials
+        | AppCredentials
+        | OAuth2ClientCredentials
+        | AppStoreCredentials
+        | UnauthCredentials
+        | CustomCredentials
+        | TbaCredentials
+    > {
```

Comment:
Not entirely sure it was on purpose, I feel like it was incorrect unless only some credentials are returned from the api

Reply:
I don't know if it was done on purpose @khaliqgant might know

Reply:
Nope, was just missing some of the recently added types but this method should actually be updated to actually return just the access token but that's for another task/issue.